### DISCUSSION TITLE
Reflex vs Theme UI

### DISCUSSION THREAD

**Discussion by estudiooka** [reflexjs/reflexjs#232] [Q&A]
I am trying to decide the framework to work with . Reflex looking very interesting.
Could you please , point out some advantages in using Reflex Instead of Theme UI it self ?
Can we use both at same time ?

Thanks.

**Comment by shadcn** [comment]
Reflexjs is built on top of theme-ui. So everything that is in theme-ui works with Reflexjs.

You can definitely use both at the same time. They both consume similar theme specs.

Here's an example with styles props (from Reflexjs) and `sx` props from theme-ui:

```ts
export function ExampleComponent() {
  return <div background="red" sx={{
    color: "body"
  }}>A div</div>
}
```

**Reply by estudiooka** [reply]
Ok, so the main diference is the nice props binding with native elements as well. 
I end up using both and also radix, really nice.
thanks..
